newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -65,6 +88,34 @@ export default function SummaryPage() {
               )}
             </Message>
           )}
+          {
+            <Modal onClose={handleMailModalClose} size="small" closeIcon open={mailModalOpen}>
+              <Modal.Header>
+                <span>E-mail participants </span>
+                <Label color="green" size="small" circular>
+                  {participantsWithEmail.length}
+                </Label>
+              </Modal.Header>
+              <Modal.Content>
+                {hasParticipantsWithoutEmail && (
+                  <>
+                    Some of your recipients do not have e-mail addresses and will not be contacted:
+                    <RecipientList recipients={participantsWithoutEmail} color="red" icon="close" />
+                    <br />
+                  </>
+                )}
+                {participantsWithEmail.length} participant
+                {participantsWithEmail.length > 1 && <>s</>} will be e-mailed:
```

Comment:
I like that you take care of pluralization but I think it would  be better to do that only once we add i18n - that'll allow doing it nicely without "hacks" like this. For now I would just always use the plural form.

If you prefer to keep it: `{participantsWithEmail.length > 1 && 's'}` would be a more concise version